Deployment failure

Latest PR #93 has been merged into the main branch but failed to deploy to the production environment due to the following error in Netlify:

```
6:59:04 PM: build-image version: 3d3c7e8b4321e2c1a54a2c4584fb46ba742b1630 (focal)
6:59:04 PM: buildbot version: d85d718afb4cfc1d31fb4837470daa2b526f928f
6:59:04 PM: Building without cache
6:59:04 PM: Starting to prepare the repo for build
6:59:04 PM: No cached dependencies found. Cloning fresh repo
6:59:04 PM: git clone --filter=blob:none https://github.com/VoiceDeck/app
6:59:04 PM: Preparing Git Reference refs/heads/main
6:59:06 PM: Starting to install dependencies
6:59:07 PM: Python version set to 3.8
6:59:07 PM: Attempting Ruby version 2.7.2, read from environment
6:59:07 PM: Using Ruby version 2.7.2
6:59:08 PM: Started restoring cached go cache
6:59:08 PM: Finished restoring cached go cache
6:59:10 PM: go version go1.19.13 linux/amd64
6:59:11 PM: Using PHP version 8.0
6:59:13 PM: Downloading and installing node v20.12.2...
6:59:13 PM: Downloading https://nodejs.org/dist/v20.12.2/node-v20.12.2-linux-x64.tar.xz...
6:59:14 PM: Computing checksum with sha256sum
6:59:14 PM: Checksums matched!
6:59:16 PM: Now using node v20.12.2 (npm v10.5.0)
6:59:16 PM: Enabling Node.js Corepack
6:59:16 PM: Started restoring cached build plugins
6:59:16 PM: Finished restoring cached build plugins
6:59:16 PM: Started restoring cached corepack dependencies
6:59:16 PM: Finished restoring cached corepack dependencies
6:59:16 PM: Installing npm packages using bun version 1.1.4
6:59:16 PM: bun install v1.1.4 (fbe2fe0c)
7:01:04 PM: Failed during stage 'Install dependencies': dependency_installation script returned non-zero exit code: 1
7:01:04 PM: make: Entering directory '/opt/build/repo/node_modules/grpc/build'
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/init.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/avl/avl.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/backoff/backoff.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/channel/channel_args.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/channel/channel_stack.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/channel/channel_stack_builder.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/channel/channel_trace.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/channel/channelz.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/channel/channelz_registry.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/channel/connected_channel.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/channel/handshaker.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/channel/handshaker_registry.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/channel/status_util.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/compression/compression.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/compression/compression_args.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/compression/compression_internal.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/compression/message_compress.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/compression/stream_compression.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/compression/stream_compression_gzip.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/compression/stream_compression_identity.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/debug/stats.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/debug/stats_data.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/http/format_request.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/http/httpcli.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/http/parser.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/buffer_list.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/call_combiner.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/cfstream_handle.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/combiner.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/endpoint.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/endpoint_cfstream.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/endpoint_pair_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/endpoint_pair_uv.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/endpoint_pair_windows.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/error.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/error_cfstream.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/ev_epoll1_linux.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/ev_epollex_linux.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/ev_poll_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/ev_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/ev_windows.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/exec_ctx.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/executor.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/fork_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/fork_windows.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/gethostname_fallback.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/gethostname_host_name_max.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/gethostname_sysconf.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/grpc_if_nametoindex_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/grpc_if_nametoindex_unsupported.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/internal_errqueue.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/iocp_windows.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/iomgr.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/iomgr_custom.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/iomgr_internal.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/iomgr_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/iomgr_posix_cfstream.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/iomgr_uv.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/iomgr_windows.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/is_epollexclusive_available.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/load_file.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/lockfree_event.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/polling_entity.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/pollset.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/pollset_custom.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/pollset_set.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/pollset_set_custom.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/pollset_set_windows.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/pollset_uv.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/pollset_windows.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/resolve_address.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/resolve_address_custom.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/resolve_address_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/resolve_address_windows.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/resource_quota.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/sockaddr_utils.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/socket_factory_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/socket_mutator.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/socket_utils_common_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/socket_utils_linux.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/socket_utils_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/socket_utils_uv.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/socket_utils_windows.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/socket_windows.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/tcp_client.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/tcp_client_cfstream.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/tcp_client_custom.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/tcp_client_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/tcp_client_windows.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/tcp_custom.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/tcp_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/tcp_server.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/tcp_server_custom.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/tcp_server_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/tcp_server_utils_posix_common.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/tcp_server_utils_posix_ifaddrs.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/tcp_server_utils_posix_noifaddrs.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/tcp_server_windows.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/tcp_uv.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/tcp_windows.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/time_averaged_stats.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/timer.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/timer_custom.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/timer_generic.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/timer_heap.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/timer_manager.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/timer_uv.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/udp_server.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/unix_sockets_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/unix_sockets_posix_noop.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/wakeup_fd_eventfd.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/wakeup_fd_nospecial.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/wakeup_fd_pipe.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/iomgr/wakeup_fd_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/json/json.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/json/json_reader.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/json/json_string.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/json/json_writer.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/slice/b64.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/slice/percent_encoding.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/slice/slice.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/slice/slice_buffer.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/slice/slice_intern.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/slice/slice_string_helpers.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/api_trace.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/byte_buffer.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/byte_buffer_reader.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/call.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/call_details.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/call_log_batch.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/channel.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/channel_init.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/channel_ping.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/channel_stack_type.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/completion_queue.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/completion_queue_factory.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/event_string.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/lame_client.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/metadata_array.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/server.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/validate_metadata.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/version.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/transport/bdp_estimator.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/transport/byte_stream.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/transport/connectivity_state.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/transport/error_utils.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/transport/metadata.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/transport/metadata_batch.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/transport/pid_controller.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/transport/static_metadata.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/transport/status_conversion.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/transport/status_metadata.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/transport/timeout_encoding.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/transport/transport.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/transport/transport_op_string.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/uri/uri_parser.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/debug/trace.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/server/secure/server_secure_chttp2.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/bin_decoder.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/bin_encoder.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/chttp2_plugin.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/chttp2_transport.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/context_list.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/flow_control.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/frame_data.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/frame_goaway.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/frame_ping.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/frame_rst_stream.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/frame_settings.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/frame_window_update.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/hpack_encoder.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/hpack_parser.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/hpack_table.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/http2_settings.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/huffsyms.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/incoming_metadata.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/parsing.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/stream_lists.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/stream_map.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/varint.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/transport/writing.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/alpn/alpn.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/http/client/http_client_filter.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/http/http_filters_plugin.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/http/message_compress/message_compress_filter.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/http/server/http_server_filter.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/http/httpcli_security_connector.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/context/security_context.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/alts/alts_credentials.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/composite/composite_credentials.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/credentials.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/credentials_metadata.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/fake/fake_credentials.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/google_default/credentials_generic.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/google_default/google_default_credentials.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/iam/iam_credentials.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/jwt/json_token.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/jwt/jwt_credentials.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/jwt/jwt_verifier.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/local/local_credentials.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/oauth2/oauth2_credentials.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/plugin/plugin_credentials.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/ssl/ssl_credentials.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/tls/grpc_tls_credentials_options.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/tls/spiffe_credentials.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/security_connector/alts/alts_security_connector.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/security_connector/fake/fake_security_connector.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/security_connector/load_system_roots_fallback.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/security_connector/load_system_roots_linux.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/security_connector/local/local_security_connector.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/security_connector/security_connector.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/security_connector/ssl/ssl_security_connector.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/security_connector/ssl_utils.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/security_connector/tls/spiffe_security_connector.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/transport/client_auth_filter.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/transport/secure_endpoint.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/transport/security_handshaker.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/transport/server_auth_filter.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/transport/target_authority_table.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/transport/tsi_error.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/util/json_util.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/surface/init_secure.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/crypt/aes_gcm.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/crypt/gsec.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/frame_protector/alts_counter.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/frame_protector/alts_crypter.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/frame_protector/alts_frame_protector.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/frame_protector/alts_record_protocol_crypter_common.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/frame_protector/alts_seal_privacy_integrity_crypter.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/frame_protector/alts_unseal_privacy_integrity_crypter.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/frame_protector/frame_handler.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/handshaker/alts_handshaker_client.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/handshaker/alts_shared_resource.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/handshaker/alts_tsi_handshaker.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/zero_copy_frame_protector/alts_grpc_integrity_only_record_protocol.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/zero_copy_frame_protector/alts_grpc_privacy_integrity_record_protocol.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/zero_copy_frame_protector/alts_grpc_record_protocol_common.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/zero_copy_frame_protector/alts_iovec_record_protocol.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/zero_copy_frame_protector/alts_zero_copy_grpc_protector.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/alts/check_gcp_environment.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/alts/check_gcp_environment_linux.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/alts/check_gcp_environment_no_op.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/alts/check_gcp_environment_windows.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/alts/grpc_alts_credentials_client_options.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/alts/grpc_alts_credentials_options.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/lib/security/credentials/alts/grpc_alts_credentials_server_options.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/handshaker/alts_handshaker_service_api.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/handshaker/alts_handshaker_service_api_util.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/handshaker/alts_tsi_utils.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/handshaker/transport_security_common_api.o
7:01:04 PM:   CC(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/handshaker/altscontext.pb.o
7:01:04 PM:   CC(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/handshaker/handshaker.pb.o
7:01:04 PM:   CC(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/alts/handshaker/transport_security_common.pb.o
7:01:04 PM:   CC(target) Release/obj.target/grpc/deps/grpc/third_party/nanopb/pb_common.o
7:01:04 PM:   CC(target) Release/obj.target/grpc/deps/grpc/third_party/nanopb/pb_decode.o
7:01:04 PM:   CC(target) Release/obj.target/grpc/deps/grpc/third_party/nanopb/pb_encode.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/transport_security.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/client/insecure/channel_create.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/client/insecure/channel_create_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/client/authority.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/client/chttp2_connector.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/backup_poller.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/channel_connectivity.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/client_channel.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/client_channel_channelz.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/client_channel_factory.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/client_channel_plugin.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/connector.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/global_subchannel_pool.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/health/health_check_client.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/http_connect_handshaker.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/http_proxy.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/lb_policy.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/lb_policy_registry.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/local_subchannel_pool.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/parse_address.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/proxy_mapper.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/proxy_mapper_registry.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/resolver.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/resolver_registry.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/resolver_result_parsing.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/resolving_lb_policy.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/retry_throttle.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/server_address.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/service_config.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/subchannel.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/subchannel_pool_interface.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/deadline/deadline_filter.o
7:01:04 PM:   CC(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/health/health.pb.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/fake_transport_security.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/local_transport_security.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/ssl/session_cache/ssl_session_boringssl.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/ssl/session_cache/ssl_session_cache.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/ssl/session_cache/ssl_session_openssl.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/ssl_transport_security.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/tsi/transport_security_grpc.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/server/chttp2_server.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/client/secure/secure_channel_create.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/server/insecure/server_chttp2.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/chttp2/server/insecure/server_chttp2_posix.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/inproc/inproc_plugin.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/transport/inproc/inproc_transport.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/lb_policy/grpclb/client_load_reporting_filter.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb_channel_secure.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb_client_stats.o
7:01:04 PM:   CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/lb_policy/grpclb/load_balancer_api.o
7:01:04 PM: make: Leaving directory '/opt/build/repo/node_modules/grpc/build'
7:01:04 PM: Failed to execute '/opt/buildhome/.nvm/versions/node/v20.12.2/bin/node /opt/buildhome/.nvm/versions/node/v20.12.2/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --library=static_library --module=/opt/build/repo/node_modules/grpc/src/node/extension_binary/node-v115-linux-x64-glibc/grpc_node.node --module_name=grpc_node --module_path=/opt/build/repo/node_modules/grpc/src/node/extension_binary/node-v115-linux-x64-glibc --napi_version=9 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v115' (1)
7:01:04 PM: node-pre-gyp info it worked if it ends with ok
7:01:04 PM: node-pre-gyp info using node-pre-gyp@0.13.0
7:01:04 PM: node-pre-gyp info using node@20.12.2 | linux | x64
7:01:04 PM: node-pre-gyp WARN Using needle for node-pre-gyp https download
7:01:04 PM: node-pre-gyp info check checked for "/opt/build/repo/node_modules/grpc/src/node/extension_binary/node-v115-linux-x64-glibc/grpc_node.node" (not found)
7:01:04 PM: node-pre-gyp http GET https://node-precompiled-binaries.grpc.io/grpc/v1.21.1/node-v115-linux-x64-glibc.tar.gz
7:01:04 PM: node-pre-gyp http 404 https://node-precompiled-binaries.grpc.io/grpc/v1.21.1/node-v115-linux-x64-glibc.tar.gz
7:01:04 PM: node-pre-gyp WARN Tried to download(404): https://node-precompiled-binaries.grpc.io/grpc/v1.21.1/node-v115-linux-x64-glibc.tar.gz
7:01:04 PM: node-pre-gyp WARN Pre-built binaries not found for grpc@1.21.1 and node@20.12.2 (node-v115 ABI, glibc) (falling back to source compile with node-gyp)
7:01:04 PM: node-pre-gyp http 404 status code downloading tarball https://node-precompiled-binaries.grpc.io/grpc/v1.21.1/node-v115-linux-x64-glibc.tar.gz
7:01:04 PM: gyp info it worked if it ends with ok
7:01:04 PM: gyp info using node-gyp@10.0.1
7:01:04 PM: gyp info using node@20.12.2 | linux | x64
7:01:04 PM: gyp info ok
7:01:04 PM: gyp info it worked if it ends with ok
7:01:04 PM: gyp info using node-gyp@10.0.1
7:01:04 PM: gyp info using node@20.12.2 | linux | x64
7:01:04 PM: gyp info find Python using Python version 3.8.10 found at "/opt/buildhome/python3.8/bin/python3"
7:01:04 PM: gyp http GET https://nodejs.org/download/release/v20.12.2/node-v20.12.2-headers.tar.gz
7:01:04 PM: gyp http 200 https://nodejs.org/download/release/v20.12.2/node-v20.12.2-headers.tar.gz
7:01:04 PM: gyp http GET https://nodejs.org/download/release/v20.12.2/SHASUMS256.txt
7:01:04 PM: gyp http 200 https://nodejs.org/download/release/v20.12.2/SHASUMS256.txt
7:01:04 PM: gyp info spawn /opt/buildhome/python3.8/bin/python3
7:01:04 PM: gyp info spawn args [
7:01:04 PM: gyp info spawn args '/opt/buildhome/.nvm/versions/node/v20.12.2/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
7:01:04 PM: gyp info spawn args 'binding.gyp',
7:01:04 PM: gyp info spawn args '-f',
7:01:04 PM: gyp info spawn args 'make',
7:01:04 PM: gyp info spawn args '-I',
7:01:04 PM: gyp info spawn args '/opt/build/repo/node_modules/grpc/build/config.gypi',
7:01:04 PM: gyp info spawn args '-I',
7:01:04 PM: gyp info spawn args '/opt/buildhome/.nvm/versions/node/v20.12.2/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
7:01:04 PM: gyp info spawn args '-I',
7:01:04 PM: gyp info spawn args '/opt/buildhome/.cache/node-gyp/20.12.2/include/node/common.gypi',
7:01:04 PM: gyp info spawn args '-Dlibrary=shared_library',
7:01:04 PM: gyp info spawn args '-Dvisibility=default',
7:01:04 PM: gyp info spawn args '-Dnode_root_dir=/opt/buildhome/.cache/node-gyp/20.12.2',
7:01:04 PM: gyp info spawn args '-Dnode_gyp_dir=/opt/buildhome/.nvm/versions/node/v20.12.2/lib/node_modules/npm/node_modules/node-gyp',
7:01:04 PM: gyp info spawn args '-Dnode_lib_file=/opt/buildhome/.cache/node-gyp/20.12.2/<(target_arch)/node.lib',
7:01:04 PM: gyp info spawn args '-Dmodule_root_dir=/opt/build/repo/node_modules/grpc',
7:01:04 PM: gyp info spawn args '-Dnode_engine=v8',
7:01:04 PM: gyp info spawn args '--depth=.',
7:01:04 PM: gyp info spawn args '--no-parallel',
7:01:04 PM: gyp info spawn args '--generator-output',
7:01:04 PM: gyp info spawn args 'build',
7:01:04 PM: gyp info spawn args '-Goutput_dir=.'
7:01:04 PM: gyp info spawn args ]
7:01:04 PM: gyp info ok
7:01:04 PM: gyp info it worked if it ends with ok
7:01:04 PM: gyp info using node-gyp@10.0.1
7:01:04 PM: gyp info using node@20.12.2 | linux | x64
7:01:04 PM: gyp info spawn make
7:01:04 PM: gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
7:01:04 PM: In file included from /usr/include/string.h:495,
7:01:04 PM:                  from ../deps/grpc/third_party/nanopb/pb.h:68,
7:01:04 PM:                  from ../deps/grpc/third_party/nanopb/pb_decode.h:9,
7:01:04 PM:                  from ../deps/grpc/src/core/ext/filters/client_channel/lb_policy/grpclb/load_balancer_api.cc:21:
7:01:04 PM: In function ‘char* strncpy(char*, const char*, size_t)’,
7:01:04 PM:     inlined from ‘grpc_grpclb_request* grpc_grpclb_request_create(const char*)’ at ../deps/grpc/src/core/ext/filters/client_channel/lb_policy/grpclb/load_balancer_api.cc:70:10:
7:01:04 PM: /usr/include/x86_64-linux-gnu/bits/string_fortified.h:106:34: error: ‘char* __builtin_strncpy(char*, const char*, long unsigned int)’ specified bound 128 equals destination size [-Werror=stringop-truncation]
7:01:04 PM:   106 |   return __builtin___strncpy_chk (__dest, __src, __len, __bos (__dest));
7:01:04 PM:       |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
7:01:04 PM: cc1plus: all warnings being treated as errors
7:01:04 PM: make: *** [grpc.target.mk:503: Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/lb_policy/grpclb/load_balancer_api.o] Error 1
7:01:04 PM: gyp ERR! build error
7:01:04 PM: gyp ERR! stack Error: `make` failed with exit code: 2
7:01:04 PM: gyp ERR! stack at ChildProcess.<anonymous> (/opt/buildhome/.nvm/versions/node/v20.12.2/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:209:23)
7:01:04 PM: gyp ERR! System Linux 5.10.210-201.855.amzn2.x86_64
7:01:04 PM: gyp ERR! command "/opt/buildhome/.nvm/versions/node/v20.12.2/bin/node" "/opt/buildhome/.nvm/versions/node/v20.12.2/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--library=static_library" "--module=/opt/build/repo/node_modules/grpc/src/node/extension_binary/node-v115-linux-x64-glibc/grpc_node.node" "--module_name=grpc_node" "--module_path=/opt/build/repo/node_modules/grpc/src/node/extension_binary/node-v115-linux-x64-glibc" "--napi_version=9" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v115"
7:01:04 PM: gyp ERR! cwd /opt/build/repo/node_modules/grpc
7:01:04 PM: gyp ERR! node -v v20.12.2
7:01:04 PM: gyp ERR! node-gyp -v v10.0.1
7:01:04 PM: gyp ERR! not ok
7:01:04 PM: node-pre-gyp ERR! build error
7:01:04 PM: node-pre-gyp ERR! stack Error: Failed to execute '/opt/buildhome/.nvm/versions/node/v20.12.2/bin/node /opt/buildhome/.nvm/versions/node/v20.12.2/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --library=static_library --module=/opt/build/repo/node_modules/grpc/src/node/extension_binary/node-v115-linux-x64-glibc/grpc_node.node --module_name=grpc_node --module_path=/opt/build/repo/node_modules/grpc/src/node/extension_binary/node-v115-linux-x64-glibc --napi_version=9 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v115' (1)
7:01:04 PM: node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/opt/build/repo/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
7:01:04 PM: node-pre-gyp ERR! stack     at ChildProcess.emit (node:events:518:28)
7:01:04 PM: node-pre-gyp ERR! stack     at maybeClose (node:internal/child_process:1105:16)
7:01:04 PM: node-pre-gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:305:5)
7:01:04 PM: node-pre-gyp ERR! System Linux 5.10.210-201.855.amzn2.x86_64
7:01:04 PM: node-pre-gyp ERR! command "/opt/buildhome/.nvm/versions/node/v20.12.2/bin/node" "/opt/build/repo/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build" "--library=static_library"
7:01:04 PM: node-pre-gyp ERR! cwd /opt/build/repo/node_modules/grpc
7:01:04 PM: node-pre-gyp ERR! node -v v20.12.2
7:01:04 PM: node-pre-gyp ERR! node-pre-gyp -v v0.13.0
7:01:04 PM: node-pre-gyp ERR! not ok
7:01:04 PM: error: install script from "grpc" exited with 1
7:01:04 PM: Error during bun install
7:01:04 PM: Failing build: Failed to install dependencies
```

--- a/bunfig.toml
+++ b/bunfig.toml
@@ -1,0 +1,4 @@
+[install]
+
+# Prevent the package manager from installing the grpc package, which we do not use. See more details in https://github.com/VoiceDeck/app/issues/95#issuecomment-2104509962
+peer = false

